docs/how-to/build-cardano-sl-and-daedalus-from-source-code.md

```diff
@@ -54,9 +54,9 @@ Two steps remain, then:
         binary-cache-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=
 
 2.  Actually building the Cardano SL node (or, most likely, simply obtaining it
-    from the IOHK's binary caches) can be performed by building the attribute `cardano-sl-node-static`:
+    from the IOHK's binary caches) can be performed by building the attribute `cardano-sl-static`:
 
-        $ nix-build -A cardano-sl-node-static --cores 0 --max-jobs 2 --no-build-output --out-link master
+        $ nix-build -A cardano-sl-static --cores 0 --max-jobs 2 --no-build-output --out-link master
```

Comment:
It looks wrong, see #2061

Reply:
No.

```
~/cardano-sl $ nix-build -A cardano-sl-node-static --cores 0 --max-jobs 2 --no-build-output --out-link master
error: attribute ‘cardano-sl-node-static’ in selection path ‘cardano-sl-node-static’ not found
```

Tested on the latest `master` 1 minute ago..

Reply:
This PR targets `develop`, so it should work in `develop`.